app/components/form-dialog.tsx

```
@@ -0,0 +1,84 @@
+    className,
+    confirmText,
+    cancelBtnText,
+    disabled = false,
+    fetcher,
+    action,
+    ...dialogProps
+  } = props
+  const navigation = useNavigation()
+
+  // Close when fetcher returns new data (new ref each response)
+  useEffect(() => {
```

Biome (2.4.4)

[error] This hook does not specify its dependency on dialogProps.onOpenChange.

(lint/correctness/useExhaustiveDependencies)